requirements.txt

```diff
@@ -23,6 +23,8 @@ Pillow==8.3.2
 sncosmo==2.5.0
 tdtax==0.1.3
 conesearch-alchemy==1.0.0
+mocpy==0.8.5
+healpix-alchemy @ git+https://github.com/skyportal/healpix-alchemy.git@cf7acbd3d0aec6fbb71a31e072e067c874de8181#egg=healpix_alchemy
```

Comment:
@mcoughlin I'm pretty confused about all the healpix-alchemy stuff... is this the version that should be pinned here? Galaxy API tests pass with this.